src/components/Layout/Header/index.tsx

```diff
@@ -4,68 +4,78 @@ import { Burger } from "./burger"
 import { CreateUser } from "lib//userTag"
 import { LogoutFunc } from "lib/loginFunc"
 import Image from "next/image"
-import icon from "public/monster.png"
+import { logo } from "public"
 import { SearchBar } from "./search"
 
 export const Header: FC = () => {
-
-
   return (
-    <header className="header flex w-full items-center sm:bg-white from-black justify-between px-2 pb6 h-20 z-10">
-      <div className="mobile-view text-white">
+    <header className="header flex w-full items-center from-black justify-between px-2 pb6 h-20 z-10">
+      <div className="mobile-view text-white flex-col">
         <CreateUser />
+        <span className="text-black">
+          {" "}
+          <SearchBar />{" "}
```

Comment:
What is empty string means 

This is not the way to write empty space or break live remove this